File: internal/adapters/lsp/servers/basedpyright/testdata/basic/fixture.py
FIXTURE_STAMP = "stamp"
fixture_counter = 0


class Bucket:
    def __init__(self, value: str) -> None:
        self.value = value

    def describe(self) -> str:
        return self.value


def make_bucket(value: str) -> Bucket:
    return Bucket(value)

File: internal/adapters/lsp/servers/phpactor/testdata/basic/fixture.php
<?php

const FIXTURE_STAMP = 'stamp';

class Bucket
{
    private string $value;

    public function __construct(string $value)
    {
        $this->value = $value;
    }

    public function describe(): string
    {
        return $this->value;
    }
}

function make_bucket(string $value): Bucket
{
    return new Bucket($value);
}

function directory_filter_target(): string
{
    return 'root';
}
